plutus-contract/src/Wallet/Emulator/ChainIndex.hs

```diff
@@ -69,15 +78,15 @@ data ChainIndexState =
 
 makeLenses ''ChainIndexState
 
-type ChainIndexEffs = '[State ChainIndexState, Writer [ChainIndexEvent]]
+type ChainIndexEffs = '[State ChainIndexState, LogMsg ChainIndexEvent]
```

Comment:
Is this right? Would this mean that e.g. if I set the log verbosity to something higher than INFO, no notifications would be processed? I'm not sure this *is* a logging effect: it's producing internal notifications that have to be handled by another part of the system.

Reply:
It is treated like a log message everywhere, we don't do any processing based on `ChainIndexEvent`. Perhaps the type just has the wrong name?

Reply:
Oh, okay, maybe I'm thinking of something else then!